module/config.mjs

```diff
@@ -1292,6 +1292,19 @@ preLocalize("armorTypes");
 
 /* -------------------------------------------- */
 
+/**
+ * Specific equipment types that modify base AC.
+ * @enum {string}
+ */
+DND5E.armorMaxDex = {
```

Comment:
Wondering if this could be merged into `DND5E.armorTypes`, but unsure what to do with the `shield` value.